tools/test/travis-ci/scancode-evaluate.py

```diff
@@ -112,25 +129,22 @@ def license_check(scancode_output_path):
             # check the next file in the scancode output
             continue
 
-        if not has_permissive_text_in_scancode_output(scancode_output_data_file['licenses']):
-            scancode_output_data_file['fail_reason'] = MISSING_PERMISSIVE_LICENSE_TEXT
-            license_offenders.append(scancode_output_data_file)
+        if not (has_permissive_text_in_scancode_output(scancode_output_data_file['licenses'])):
+            scanned_file_content = get_file_text(scancode_output_data_file)
+
+            if not (scanned_file_content is not None and has_binary_license(scanned_file_content)):
```

Comment:
Wouldn't the following do the same?
```suggestion
            if not (scanned_file_content and has_binary_license(scanned_file_content)):
```